### ISSUE TITLE
Unable to install cyvlfeat via conda

### ISSUE THREAD

**Issue by Azurestarry** [menpo/cyvlfeat#58]
It seems to be a bit weird that "conda install -c conda-forge cyvlfeat" does not work on my Anaconda prompt, and I'm not sure whether it's my fault (lol). Here is the screenshot, thx!

![20200622155248](https://user-images.githubusercontent.com/30016169/85263001-21525680-b4a1-11ea-9b5a-ec9afd8caf6b.png)

**Comment by hubutui** [comment]
It seems that `cyvlfeat` is not built for windows in conda-forge channel, you could use the menpo channel. @patricksnape Any particular reason to change from menpo channel to conda-forge channel?

**Comment by patricksnape** [comment]
For a small history: the menpo channel pre-existed conda-forge and is what we used to use to distribute all of our software. However, conda-forge is a much better and bigger implementation of a similar concept to the menpo channel and should be preferred. Not least  because the menpo channel is now functionally dead.

However, in this particular case the issue is that Windows builds are currently disabled due to the complexity in trying to build them. A long time ago I decided that Windows builds were just too hard to maintain as an Open Source developer. If someone wants to work and pulling across the azure builds from conda-forge then we can maybe think about reenabling Windows here - but you must know that Windows builds are really hard to maintain sometimes!
